component/main.jsonnet

```
@@ -4,6 +4,7 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.blackbox_exporter;
+local instance = inv.parameters._instance;
 
 // Define outputs below
 {
```

Did you mean to add the following here?

```suggestion
{
  '00_namespace': kube.Namespace(params.namespace)
```

---

What happens, if the namespace already exist?
How can I best handle this, if I don't want to touch an already existing namespace?

---

Easiest way is to have an additional parameter to control whether the namespace should be created or not.

---

Thanks for the suggestion. Applied a slightly different config that takes optional namespace creation into consideration.